the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/bulkstatus/BulkStatusController.java

```diff
@@ -0,0 +1,156 @@
+package gov.va.api.health.bulkfhir.service.controller.bulkstatus;
+
+import gov.va.api.health.autoconfig.logging.Loggable;
+import gov.va.api.health.bulkfhir.api.bulkstatus.PublicationFileStatusResponse;
+import gov.va.api.health.bulkfhir.service.status.StatusEntity;
+import gov.va.api.health.bulkfhir.service.status.StatusRepository;
+import gov.va.api.health.ids.api.IdentityService;
+import gov.va.api.health.ids.api.IdentityService.LookupFailed;
+import gov.va.api.health.ids.api.ResourceIdentity;
+import gov.va.api.health.ids.client.IdEncoder.BadId;
+import java.time.Instant;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@Validated
+@RestController
+@Loggable
+@RequestMapping(
+  value = {"bulk"},
+  produces = {"application/json"}
+)
+public class BulkStatusController {
+
+  /** The list of valid values for the Accept header for an $export request. */
+  private static List<String> VALID_ACCEPT_HEADER_VALUES =
+      List.of("application/fhir+json", "application/json");
```

Comment:
Should `application/json+fhir` be here as well?

Reply:
We can add that if we want, don't have a strong opinion either way.